automation/xpath

```diff
@@ -0,0 +1,29 @@
+XPath Project
+
+1. Page: /text-box. Write the xpath that will return the �Full Name� textbox using the ID attribute.
+Hint: This element has the �input� tag.
+$x("//input[@id='userName']")
+
+2. Page: /buttons. Write the xpath that will return all elements on the page with the �button� tag.
+$x("//button")
+
+3.Page: /webtables. Write the xpath that will return the div with text of �Vega�. Ensure to narrow
+the scope of your xpath to the ReactTable and ensure that your xpath is limited to returning
+cells only. Cells in this table have the class �rt-td�. Hint: You must use �Vega� in your xpath.
+$x("//div[text()='Vega'][@class='rt-td']")
```

Comment:
Search within the scope of the table
//div[@class='ReactTable -striped -highlight']//div[text()='Vega']